macro/src/lib.rs

```diff
@@ -39,41 +39,59 @@
 //! This will replace the generated type and any usages with the specified type at the `use` import.
```

Comment:
I can resolve this conflict when it merges, but I've also re-done these macro docs so idealls you'd roll back any changes to them for the new guide :)

Reply:
okay, docs rolled back